src/webpack/local.index.js

```diff
@@ -1,6 +1,9 @@
 import BorosTcf from '../main'
 
-BorosTcf.init()
+const reporter = {
```

Comment:
reporter shouldn't be only a function? not an object with method notify

Reply:
ups, I've changed to a function in a last commit and didn't test the local demo, thanks!